doc/md/tutorial/_hello-holos/script-02-hello-holos/tree.txt

```diff
@@ -0,0 +1,17 @@
+.
+|-- components
+|   `-- podinfo
```

Comment:
I don't know why it's creating backticks here instead of `└──`. I've tried running `tree -A` and `tree -S` to use different character sets, and tried `tree -n` to turn off colorization, but it doesn't solve the problem.